### PR TITLE
added sidebar buttons on pledged pathway page

### PR DESCRIPTION
**Pull request by malynda**
addendum to Issue #171

### CODE REVIEW

**Comment by malynda** [comment]
Undo button wasn't included in the wireframe, but thought it might be needed. Looks ugly so that we remember to change the icon later :)

**Comment by malynda** [comment]
@stenington All functionality for Issue #171 has been included! Old PR closed since everything is now included here.

File: clientapp/styles/pathway-editor.scss [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+  }
+}
+
+.undo-btn{
+  background: url(/static/pathway/icon-view-similar.svg) 0 0 no-repeat;
+  &.active{
+    background: url(/static/pathway/icon-view-similar-active.svg) 0 0 no-repeat;
+  }
+}
+
```
Comment (ZeeJab) [inline clientapp/styles/pathway-editor.scss:38]: Can these buttons be moved to the button.scss file?

File: clientapp/templates/includes/editor.html [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+          <li><a href="#" class="js-view-similar view-similar-btn"></a></li>
+          <li><a href="#" class="js-add-badge add-badge-btn" data-stack="addPanel" ></a></li>
+          <li><a href="#" class="js-add-storybit add-storybit-btn"></a></li>
+          <li><a href="#" class="remove-items-btn js-delete" data-toggle="off"></a> </li>
+          <li> <a href="#" class="undo-btn js-undo">Undo</a> </li>
+        </div>
+      </div>
+    {% endif %}
+    <div class="small-12 medium-8 medium-centered columns">
+      <canvas class="editor" width="1000"></canvas>
```
Comment (ZeeJab) [inline clientapp/templates/includes/editor.html:14]: if this is a min width that is needed please added it to the css as `min-width: 100px;`

File: clientapp/templates/includes/add-item.html [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
+                <div for="{{_id}}" > </div>
+            </label>
+        </div>
+        <div class="data">
+            <h5>{{title}}</h5>
+            {% for tag in tags %}
+            <span class="label achievement-tag">#{{tag}}</span>
+            {% endfor %}
+            <div>
+                <img src="/static/sbux.jpg" width=50>
```
Comment (ZeeJab) [inline clientapp/templates/includes/add-item.html]: if this is a min width that is needed please added it to the css as `min-width:`

File: clientapp/views/includes/editor.js [hunk 4]
```diff
@@ -62,6 +70,36 @@ module.exports = HumanView.extend({
...
+    }else{
+      btn.addClass('active');}
+    evt.preventDefault();
+  },
+  addStorybits: function(evt){
+    var btn= $(evt.target);
+    if(btn.hasClass('active')){
+      btn.removeClass('active');
+    }else{
+      btn.addClass('active');}
```
Comment (ZeeJab) [inline clientapp/views/includes/editor.js]: I think this there's a lot of opportunity  in this file to refactor this in a way that is reusable? @stenington what are your thoughts?

File: clientapp/styles/pathway-editor.scss (hunk 1 above)
Comment (malynda) [inline clientapp/styles/pathway-editor.scss:38]: Done

File: clientapp/views/includes/editor.js (hunk 4 above)
Comment (malynda) [inline clientapp/views/includes/editor.js]: Refactored!

**Comment by malynda** [comment]
@zzzndr do we have an asset for an "undo" button on this sidebar in the pathway editor? (wasn't a part of the wireframes) https://www.dropbox.com/s/145fmkwd145j9jw/Screenshot%202014-04-30%2016.23.14.png

**Comment by chloeatplay** [comment]
Good catch @zzzndr we should add one.

**Comment by zzzndr** [comment]
good point!
will create a button, can you use a placeholder for now?

**Comment by zzzndr** [comment]
here is a link to the proposal for an undo button (:
https://redpen.io/cg7ab5e00161b4b294

**Comment by malynda** [comment]
@zzzndr Beautiful! I have a lousy looking placeholder so that I remember to change it to something pretty later :)

**Pull request closed without merging**